source/MenuPanel.cpp

```diff
@@ -61,7 +62,22 @@ MenuPanel::MenuPanel(PlayerInfo &player, UI &gamePanels)
 	SetIsFullScreen(true);
 
 	if(mainMenuUi->GetBox("credits").Dimensions())
-		credits = Format::Split(Files::Read(Files::Resources() + "credits.txt"), "\n");
+	{
+		for(const auto &source : GameData::Sources())
+		{
+			auto credit = Format::Split(Files::Read(source + "credits.txt"), "\n");
+			if((credit.size() > 1) || (credit.front() != ""))
+			{
+				credits.push_back("######################");
+				credits.push_back("");
+				credits.insert(credits.end(), credit.begin(), credit.end());
+				credits.push_back("");
+				credits.push_back("######################");
+				credits.push_back("");
+				credits.push_back("");
```

Comment:
Not a fan of the look of the lines of hashtags to break up credits. I'd rather just have a chunk of empty space.
```suggestion
				credits.insert(credits.end(), credit.begin(), credit.end());
				credits.insert(credits.end(), 5, "");
```

Reply:
Alright, removed the hastags.